packages/db/src/executions.ts

```diff
@@ -0,0 +1,71 @@
+import Knex from 'knex';
+
+import { ExecutionRecord } from '@cumulus/types/api/executions';
+import { PostgresExecution } from './types';
+// TODO move to common location
```

Comment:
I'm thinking these helpers should be on the new models. Does it seem reasonable to pull in the Postgres `AsyncOperation`, `Collection`, and `Execution` models here? Alternatively some common utils or shared space would do.

Reply:
duplicate comment